core/src/main/java/io/jenkins/pluginhealth/scoring/probes/RenovateProbe.java

```diff
@@ -0,0 +1,31 @@
+package io.jenkins.pluginhealth.scoring.probes;
+
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+/*
+* Looks for Renovate bot configuration in a plugin.
+* Extends `AbstractDependencyBotConfigurationProbe` class to implement most of the features.
+*
+* Renovate makes PRs to update dependencies in the plugin.
+*/
+@Component
+@Order(RenovateProbe.ORDER)
```

Comment:
inline and use directly `AbstractDependencyBotConfigurationProbe.ORDER`?

Reply:
What do you mean by inline here?

Reply:
What I mean is that in your `@Order` annotation you use a variable that is just assigning another variable with no modification.

You could directly inline `AbstractDependencyBotConfigurationProbe.ORDER` in the annotation like `@Order(AbstractDependencyBotConfigurationProbe.ORDER)` instead of assigning it to a variable that brings no added value.